cpp/src/cylon/table.cpp

```diff
@@ -310,6 +310,11 @@ Status Sort(std::shared_ptr<cylon::Table> &table, const std::vector<int32_t> &so
 
 Status Sort(std::shared_ptr<cylon::Table> &table, const std::vector<int32_t> &sort_columns,
             std::shared_ptr<cylon::Table> &out, const std::vector<bool> &sort_direction) {
+  // if single index sort is passed
```

Comment:
Should we call this function from single index sort instead and remove cylon::util::SortTable?

Reply:
you mean bring the sort table logic under this method?